tests/components/test_base_defaults_component.py

```diff
@@ -189,11 +189,11 @@ def test_multiple_generations(
     def test_env_var_substitution(
         self, config: KpopsConfig, handlers: ComponentHandlers
     ):
-        ENV["pipeline_name"] = str(DEFAULTS_PATH)
+        ENV["pipeline_name"] = DEFAULTS_PATH.as_posix()
         component = EnvVarTest(config=config, handlers=handlers)
 
-        assert component.name == str(
-            DEFAULTS_PATH
+        assert (
+            Path(component.name or "") == DEFAULTS_PATH
```

Comment:
we should remove the `name` override in `EnvVarTest`, it should never be `None`

Reply:
`name` is defined in `PipelineComponent`, so it is not inherited from `BaseDefaultsComponent` by `EnvVarTest`. I could have a Pyright ignore comment instead of the `or`?

Reply:
Ah, true. Then please put a prior `assert component.name`